### PR TITLE
Update Iris Exclusive Features through 1.6.13

### PR DESCRIPTION
**Pull request by ninjamike1211**
Updated all Iris exclusive uniforms, properties, and features through Iris 1.6.13.
Also updated the code snippets used for custom images, as they didn't really make sense in context.

### CODE REVIEW

**Comment by ninjamike1211** [comment]
Because of the way I did my commits I ended up making this one PR, if it would be better as multiple PRs I can do that too.

File: iris-features.md [hunk 1]
```diff
@@ -240,6 +412,33 @@ There are some new ID's that can be detected alongside items and armor:
...
+
+*You can use `*` as a value to fallback all dimensions.*
+
+The following example sets the shaders for the vanilla Nether dimension to the `netherShaders` folder:
+
+`dimension.netherShaders = minecraft:the_nether`
+
+# Color Spaces (Iris 1.6.4)
+
+Iris 1.6.4 added support for additional color spaces beyond sRGB (DCI_P3, Display P3, REC2020, and Adobe RGB). This allows support for outputting to HDR displays.
```
Comment (IMS212) [inline iris-features.md:434]: Iris does not allow output to HDR displays; this is just a tone mapping operation.

**Review by IMS212 — CHANGES_REQUESTED** [review]
Thanks! Only one thing.

File: iris-features.md (hunk 1 above)
Comment (ninjamike1211) [reply iris-features.md]: Great, I probably should have played around with that a bit more before making that conclusion. Let me know if the change is accurate. Thanks for getting back so quickly!

**Pull request merged**